phpunit.xml.dist

```diff
@@ -1,23 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
          bootstrap="./vendor/autoload.php"
          colors="true"
          failOnIncomplete="true"
          failOnWarning="true"
          failOnRisky="true"
->
-    <coverage>
+         cacheDirectory=".phpunit.cache">
```

Comment:
Do we need to configure this? Would keep this on default if possible.

To make switching between 2.5 <-> 2.6 easier for this who don't have such files in there global gitignore we should add the directory also to the `.gitignore` file of the `2.5` branch. Can you target a merge request for it.

Reply:
We could keep it but it's a deprecated attribute in phpunit 10 and will be removed in 11.

https://github.com/sebastianbergmann/phpunit/issues/4599
https://github.com/sebastianbergmann/phpunit/issues/4600

Reply:
Then I would remove the config and just add the new directory to the .gitignore file

Reply:
I just saw that the cache file is already in the `.gitignore`